src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -159,7 +159,7 @@ private function replaceClassKeyword(Tokens $tokens, $classIndex)
         $classBeginIndex = $classEndIndex;
         while ($tokens[--$classBeginIndex]->isGivenKind(array(T_NS_SEPARATOR, T_STRING)));
         ++$classBeginIndex;
-        $classString = $tokens->generatePartialCode($classBeginIndex, $classEndIndex);
+        $classString = ltrim($tokens->generatePartialCode($classBeginIndex, $classEndIndex), '\\');
```

Comment:
Looks like a good idea, I wouldn't worry about the extra variable.
I'm not sure about the fixed +1 offset, please add this case to the tests:

```php
<?php

class A
{
}

echo \/**/A::class;
echo \/**/A::
//
class;

echo \/**/A::#123
//
class;
```

https://3v4l.org/be3T8